app/javascript/bundles/UI/components/Alert/Alert.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import css from './Alert.scss';
+import classnames from 'classnames';
+
+var types = ['standard', 'sky', 'fire', 'sun', 'sea'];
```

Comment:
I like the easter egg but this is not standard at all 😅

Reply:
If you're talking about the variant names, those are the official palette color names :rofl: 

I suggest moving onto this notation or a similar one:

standard: sky
error: fire
warning: sun
success: sea